woocommerce/woocommerce
# Add --force to test scaffold

This PR adds an optional `--force` parameter to the test scaffold that allows to run the test scaffold, overriding local files without prompting a confirmation to the user if the files already exist.

Closes #31455

### How to test:

1. Run `cd plugins/woocommerce`
2. Run `pnpm install`
3. Rename the tests folder: `mv tests tests-x`
4. Run `pnpx wc-e2e install`
5. This should create
```
tests/e2e/config/jest.config.js
tests/e2e/config/jest.setup.js
tests/e2e/docker/initialize.sh
```
6. Run `pnpx wc-e2e install --force`
7. This should re-create the files above without prompting a confirmation

### Testing that the test files are created without a prompt:

8. As a continuation to the tests above, run `pnpx wc-e2e install @woocommerce/e2e-core-tests`
9. This should create many test files under the folder `tests/e2e/specs`
10. Now run `pnpx wc-e2e install @woocommerce/e2e-core-tests --force`
10. This should re-create the test files inside the `tests/e2e/specs` folder without prompting a confirmation
